tests/ha/pacemaker_cts_cluster_exerciser.pm

```diff
@@ -34,6 +35,12 @@ sub run {
 
     zypper_call 'in pacemaker-cts';
     save_screenshot;
+    # Get package version
+    my $pacemaker_cts_package_version = script_output("rpm -q --qf '%{VERSION}\n' pacemaker-cts");
+    # Compare package version
+    if (package_version_cmp($pacemaker_cts_package_version, '2.1.6') >= 0) {
```

Comment:
"CTSlab.py" was renamed to "cts-lab" since '2.1.6', Currently version used in 15-SP6 is  '2.1.7' so. It works for  >=2.1.6.
(15SP5 still using 2.1.5 I offered VR above)

Reply:
"CTSlab.py" was renamed to "cts-lab" since '2.1.6', Currently version used in 15-SP6 is  '2.1.7' so. It works for  >=2.1.6.
(15SP5 still using 2.1.5 I offered VR above)

Reply:
In https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18684 used different syntax like
```
if (check_version('>=2.1.6', $pacemaker_version)) {
```

Reply:
The "check_version" is good too.